### PR TITLE
PSD docs update

### PR DESCRIPTION
**Pull request by jacopok**
This is a minimal update to the documentation regarding analytical PSDs from `LALsimulation`, addressing the issue raised in https://github.com/gwastro/pycbc/issues/3938.

### CODE REVIEW

File: pycbc/psd/analytical.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Provides reference PSDs from LALSimulation.
+
+More information about how to use these can be found in the
+guide about `Analytic PSDs from lalsimulation`_.
```
Comment (spxiwh) [inline pycbc/psd/analytical.py:20]:
```suggestion
guide about :ref:`Analytic PSDs from lalsimulation`.
```

**Comment by jacopok** [comment]
Thanks for the syntax check! Though, I wonder why the way I wrote it did not work: both versions should (?) be supported by sphinx.

**Comment by spxiwh** [comment]
No idea! ... I note that you can do `python setup.py build_docs` to test this locally, but beyond that, I find sphinx mostly a trial and error type of thing!

Thanks again for the contribution. I want to look closer at the normalization issue in the other PR before approving that, and will try and get to that soon!

**Review by spxiwh — APPROVED** [review]

**Comment by jacopok** [comment]
I see! I actually tried to build the docs locally as you suggest but ran into errors doing it plainly; I saw the [documentation](https://pycbc.org/pycbc/latest/html/build_gh_pages.html) but it looked like a great deal of effort for a simple change so I did not follow it.

**Pull request merged**